packages/uniforms-bootstrap3/__tests__/AutoFields.tsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import { AutoField } from 'uniforms-antd';
 import { AutoFields } from 'uniforms-bootstrap3';
```

Comment:
It should use `AutoField` from the `uniforms-bootstrap3` theme. Same for the others.